src/components/select/select.tsx

```diff
@@ -70,7 +70,7 @@ export class Select implements LabelableComponent, FormComponent, InteractiveCom
   @Prop({ reflect: true }) name: string;
 
   /**
-   * When true, the component must have a value in order for the form to submit.
+   * When `true`, the component must have a value in order for the form to submit.
```

Comment:
Should "value" be in back ticks, since this one is referring to the property? I don't know with this one and some of the "open" ones, it could go either way really.

Reply:
Similar to above with `popover`, since its a reference to the component having a value, rather than a specific reference to the property - leaving it as value (without backticks).